k8s/k8s.go

```diff
@@ -858,6 +859,22 @@ func (k *k8sOps) SetConfig(config *rest.Config) {
 	k.client = nil
 }
 
+// SetRemoteConfig takes the path to a kubeconfig file
+// and then internally calls SetConfig to set it
+func (k *k8sOps) SetRemoteConfig(kubeconfigPath string) error {
+	if kubeconfigPath == "" {
+		k.SetConfig(nil)
+		return nil
+	}
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfigPath)
+	if err != nil {
+		return err
+	}
```

Comment:
Why not just call this from your code and then call a SetConfig

Reply:
@piyush-nimbalkar I wanted to see what people would think if we added this method directly to sched-ops.  Then any client of sched-ops, such as stork, spawn, torpedo, etc. could call this method, rather than having to re-implement the logic in each client.

Reply:
Rename to `SetConfigFromPath`. That's more appropriate than remote.